lib/player.js

```diff
@@ -131,12 +131,17 @@ Player.prototype = _.extend({
             this.emit('timechange', time)
         }
 
-        if(data.indexOf('ANS_LENGTH') !== -1 && data.indexOf('ANS_VO_FULLSCREEN') !== -1 && data.indexOf('ANS_SUB_VISIBILITY') !== -1) {
-            this.setStatus({
-                duration: parseFloat(data.match(/ANS_LENGTH=([0-9\.]*)/)[1]),
-                fullscreen: (parseInt(data.match(/ANS_VO_FULLSCREEN=([01])/)[1]) === 1),
-                subtitles: (parseInt(data.match(/ANS_SUB_VISIBILITY=([01])/)[1]) === 1)
-            });
+        if(data.indexOf('ANS_LENGTH') !== -1 || data.indexOf('ANS_VO_FULLSCREEN') !== -1 || data.indexOf('ANS_SUB_VISIBILITY') !== -1) {
```

Comment:
I would remove it but there still needs to be a check for calling the setStatus so it isn't called when nothing is changed